packages/components/input/src/input.vue

```diff
@@ -312,7 +312,10 @@ const suffixVisible = computed(
 
 const [recordCursor, setCursor] = useCursor(input)
 
+const isElHidden = () => textarea.value?.offsetParent === null
+
 useResizeObserver(textarea, (entries) => {
+  !isElHidden() && onceInitSizeTextarea()
```

Comment:
May be better to check `autosize` first to avoid useless reflow.

Reply:
resolved~

Reply:
LGTM